android-components/components/browser/state/src/test/java/mozilla/components/browser/state/engine/middleware/SessionPrioritizationMiddlewareTest.kt

```diff
@@ -64,7 +97,7 @@ class SessionPrioritizationMiddlewareTest {
     }
 
     @Test
-    fun `GIVEN a previous selected tab with priority DEFAULT WHEN selecting and linking a new tab THEN update the previous tab to DEFAULT and the new one to HIGH`() {
+    fun `GIVEN a previous selected tab with priority DEFAULT WHEN selecting and linking a new tab THEN update the new one to HIGH and the previous tab based on if it contains form data`() = runTestOnMain {
```

Comment:
nit/out of curiosity: does this mean that we can have 2 HIGH prio tabs/sessions at the same time?

Reply:
Hey Kaya, yes it would be possible to have multiple high priority tabs. Any background tab with form data may have a high priority for a certain time.